include/btllib/nthash.hpp

```diff
@@ -1541,8 +1665,8 @@ BTLLIB_NTHASH_INIT(NtHash,
                           posN,
                           hashes_vector.data()), )
 BTLLIB_NTHASH_ROLL(NtHash,
-                   ntmc64(seq[pos - 1],
```

Comment:
Was this an error before the pull request?

Reply:
Nope. I just re-ordered what was going on in BTLLIB_NTHASH_ROLL. Initially, it was `++pos` and then `NTHASH_CALL`, but I needed it to be the other way around so I had to change this to `pos - 1` for things to work.

Reply:
If it's not clear what I mean, you changed this line to `ntmc64(seq[pos],`, I'm just curious if `pos - 1` was incorrect before.